org_status/org_status.py

```diff
@@ -137,4 +160,23 @@ def main():
 
         org_host = Host(token, org, verbose=args.verbose)
         org_status = aggregate_org_status(org_host, threads=args.threads)
+
+        if args.export_orgs:
+            export_data = encode_repo_list(org_status, args.format)
+            export_file = f'{getcwd()}/{args.export_orgs}'
+
+            write_format_to_file(export_data, export_file)
+
         present_status(org_status, args.no_color)
+
+
+def write_format_to_file(export_data, export_file):
```

Comment:
Should be above `main()` and call it `write_data_to_file(encoded_data, filename)`.